index.bs

```diff
@@ -1566,12 +1566,35 @@ for each [=sensor types=]:
     especially when doing so would not make sense.
 -   A set of [=identifying parameters=]. TODO: replace that by an abstract operation.
 
-
 <h3 id="permission-api">Extending the Permission API</h3>
 
-Provide guidance on how to extend the Permission API [[PERMISSIONS]]
-for each [=sensor types=].
+{{Sensor}} interface for concrete [=sensor=] must protect it's [=sensor reading|reading=]
```

Comment:
An implementation of the {{Sensor}} interface for each [=sensor types=] must protect its